substrate/frame/src/lib.rs

```diff
@@ -233,6 +233,7 @@ pub mod runtime {
 		pub use sp_block_builder::*;
 		pub use sp_consensus_aura::*;
 		pub use sp_consensus_grandpa::*;
+		pub use sp_genesis_builder::*;
```

Comment:
IMO we should directly pull in the `Result` here as `GenesisBuilderResult`. Maybe directly rename it in the crate to make it even more obvious.

Reply:
> IMO we should directly pull in the `Result` here as `GenesisBuilderResult`. Maybe directly rename it in the crate to make it even more obvious.

Exported it from the `frame` crate but didn't rename in `sp-genesis-builder` to avoid a breaking change.